css/style.css

```diff
@@ -0,0 +1,56 @@
+/**{
+    margin: 0;
+    padding:0;
+    box-sizing: border-box;
+    background-color: beige;
+  }
+  */
+  #navbar {
+    position: fixed;
+    min-width: 200px;
+    top: 0px;
+    left: 0px;
+    width: 300px;
+    height: 100%;
+    border-right: solid;
+    border-color: black;
+  }
+  body{
+    margin-left: 320px;
+    background-color: bisque;
+    font-family: arial;
+    min-width: 200px;
+  }
+  #navbar{
+    text-align:center;
+  }
+
+  section header{
+    font-size: larger;
+    font-weight: bolder;
+  }
+#Main_Tittle{
+    text-align: center;
+  }
+@media only screen and (max-width: 800px) {
+    #main-doc {
+      margin-left: -10px;
+    }
+    #navbar{
+        font-size: small;
+        color:aqua;
+        position: absolute;
+        height: 200px;
+        width: 100%;
+    }
+    body {
```

Comment:
You could use a class attribute for multiple elements as CSS Classes will help you stylize HTML elements quickly. Moreover, you can avoid repeating the same code for each HTML element that uses the same styling. Hence, the amount of CSS code to use is reduced, making it more readable and easier to debug.

- [ ] Kindly Use just one block of code for your body and navbar element